ee/billing/test/test_quota_limiting.py

```diff
@@ -31,12 +31,12 @@ class TestQuotaLimiting(BaseTest):
     def setUp(self) -> None:
         super().setUp()
         self.redis_client = get_client()
-        self.redis_client.delete(f"{QuotaLimitingCaches.QUOTA_LIMITER_CACHE_KEY}.{QuotaResource.EVENTS}")
-        self.redis_client.delete(f"{QuotaLimitingCaches.QUOTA_LIMITER_CACHE_KEY}.{QuotaResource.RECORDINGS}")
-        self.redis_client.delete(f"{QuotaLimitingCaches.QUOTA_LIMITER_CACHE_KEY}.{QuotaResource.ROWS_SYNCED}")
-        self.redis_client.delete(f"{QuotaLimitingCaches.QUOTA_LIMITING_SUSPENDED_KEY}.{QuotaResource.EVENTS}")
-        self.redis_client.delete(f"{QuotaLimitingCaches.QUOTA_LIMITING_SUSPENDED_KEY}.{QuotaResource.RECORDINGS}")
-        self.redis_client.delete(f"{QuotaLimitingCaches.QUOTA_LIMITING_SUSPENDED_KEY}.{QuotaResource.ROWS_SYNCED}")
+        self.redis_client.delete(f"@posthog/quota-limits/.events")
+        self.redis_client.delete(f"@posthog/quota-limits/.recordings")
+        self.redis_client.delete(f"@posthog/quota-limits/.rows_synced")
+        self.redis_client.delete(f"@posthog/quota-limiting-suspended/.events")
+        self.redis_client.delete(f"@posthog/quota-limiting-suspended/.recordings")
+        self.redis_client.delete(f"@posthog/quota-limiting-suspended/.rows_synced")
```

Comment:
are the additional dots (`/.events` instead of `/events`) intentional here? What are they for?

Reply:
I don't know why it's like that, it was that way originally so I just kind of assumed it was needed. But I think you are right it should be removed, tests still pass without.